calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_foreground.jpg" alt="radar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Radar background pointcloud.</p></td>
+    <td><p style="text-align: center;">Radar foreground pointcloud (shown in purple lines). <br>The pointcloud appears as lines due to the radar's lack of elevation data.</p></td>
+  </tr>
+</table>
+
+### Add lidar-radar pair
+
+After the background model has been extracted, the user can carry the radar reflector (with the tripod) and place it in front of the radar sensor. Once the reflector is positioned, the user should step a few steps away from it. In the tutorial rosbag, the user will see that both the human and the radar reflector (with tripod) are identified as foreground objects in the image below.
+
+In the image, the colored points represent different lidar foreground clusters. The purple lines indicate radar foreground detections, appearing as lines due to the radar's lack of elevation data, making the z-axis unknown. The blue point is the estimated center of the radar reflector derived from the lidar pointcloud. The blue points did not appear on the human figure because the calibrator filters out clusters where the highest point in the cluster exceeds the specified threshold.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/add1.jpg" alt="add1" width="300" height="300">
+</p>
+
+When the purple line connects the purple point (the radar estimation of the reflector) and the blue point (the lidar estimation of the reflector), the user can press the `Add lidar-radar pair` button to register them as a pair.
+
+Afterward, if the pair that the user added converges, it will become a converged pair, which will then be used for calibration. Additionally, the colors of the markers will change: the white point indicates the lidar estimation, the red point marks the initial radar estimation, and the green point signifies the calibrated radar estimation."
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/add2.jpg" alt="add2" width="300" height="300">
+</p>
+
+As described in the [Step 3: Matching and filtering](../../marker_radar_lidar_calibrator/README.md#step-3-matching-and-filtering) in the general documentation, we rely on the initial calibration to pair each lidar detection with its closest radar detection, and vice versa. Below, we show examples of good and bad initial calibration.
+
+In both images, the calibrator detects two potential reflectors: a human and a radar reflector (located closer to the bottom of the image). In a good initial calibration image, the blue point from lidar detection will correctly match the radar detection, as they are the closest to each other. However, in the image of bad initial calibration, the blue point will incorrectly match the radar detection of the human.
```

Comment:
This is not well explained. There are two radar detections and one lidar detection

Reply:
`In a good initial calibration image`?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/d60a4f4507f89bcec49c7880a4dc3d7b1639eb63